edx_credentials_themes/templates/wharton/credentials/programs/c8f689f4-8418-41bd-a73b-64c1459c1bb2/certificate.html

```diff
@@ -0,0 +1,7 @@
+{% extends "../base.html" %}
+{% load staticfiles %}
+
+{% block program_title %} Architecture and Systems Engineering: Models and Methods to Manage Complex Systems {% endblock %}
+{% block start_date %}March 20, 2017{% endblock %}
+{% block end_date %}August 7, 2017{% endblock %}
```

Comment:
@ibrahimahmed443 
We just want to display the date the certificate was issued (see https://openedx.atlassian.net/browse/WL-1401) not the start date and end date

Reply:
@bill-filler I am going to remove these files anyway. These files were copied from MIT and they are for specific programs. Just having /professional-certificate/certificate.html is sufficient for our case. The way I have dealt with certificate date is here: https://github.com/edx/credentials-themes/pull/31/files#diff-51d8d14bf95a7bc76a20fc8cb27f0f03R39